spec/controllers/assets_controller_spec.rb

```diff
@@ -25,13 +25,27 @@
         expect(assigns(:asset).file.path).to match(/asset\.png$/)
       end
 
+      it 'stores access_limited as access_limited_user_ids on asset' do
```

Comment:
Do we need to cope with the weird empty-string-array case too?

Reply:
Added test case